app/src/main/java/org/oppia/app/topic/TopicFragment.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.app.topic
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that contains tabs for Topic. */
+class TopicFragment : InjectableFragment() {
+  @Inject lateinit var topicFragmentController: TopicFragmentController
```

Comment:
Nit: ``TopicFragmentPresenter``. Ditto elsewhere in this PR.

Reply:
Done. Changed `Controller` to `Presenter` everywhere in activity and fragment